### ISSUE TITLE
Add `first-line` except-option for `double-slash-comment-empty-line-before` 

### ISSUE THREAD

**Issue by thasmo** [kristerkari/stylelint-scss#100]
Using
```json
"scss/double-slash-comment-empty-line-before": ["always", {"except": [
	"first-nested"
], "ignore": [
	"between-comments",
	"stylelint-commands"
]}]
```
still gives errors if the comment is on line 1 of a file.

It would be nice to have an `exclude` or `ignore` option for this.

**Comment by thasmo** [comment]
Using stylelint's option `comment-empty-line-before` does not have this problem and is just working when using
```json
"comment-empty-line-before": ["always", {"except": [
	"first-nested"
], "ignore": [
	"after-comment",
	"stylelint-commands"
]}]
```

**Comment by thasmo** [comment]
The error gets actually thrown by `"scss/double-slash-comment-inline": "never"`.

**Comment by dryoma** [comment]
Could you please provide a code sample that the settings above give warnings for, as well as the warnings themselves?

**Comment by thasmo** [comment]
Rule:
```
"scss/double-slash-comment-inline": "never"
```

Error:
```
source/style/theme/default/_variable.scss
1:1  ×  Unexpected inline //-comment   scss/double-slash-comment-inline
```

Code:
```scss
// _variable.scss

$color: tomato;
```

**Comment by thasmo** [comment]
The rule `double-slash-comment-empty-line-before` doesn't influence this actually, I found.

**Comment by LoChanHi5** [comment]
I also have the same issue that @thasmo described in https://github.com/kristerkari/stylelint-scss/issues/100#issuecomment-279647041 but only for comments that appear in the _first_ line of the file.

**Comment by dryoma** [comment]
Fixed in 1.4.2 .Thanks for reporting!

**Comment by Lesechka** [comment]
Give me an advice please! 
I have no idea with work of rule "scss/double-slash-comment-empty-line-before": "never".
If I type command 'npm run lint-style' I get mistakes: 
src/styles/scss/_typography.scss
 30:5  ×  Unexpected empty line before comment   scss/double-slash-comment-empty-line-before
 35:5  ×  Unexpected empty line before comment   scss/double-slash-comment-empty-line-before.
But after command 'npm run fix-style' I get nothing.  
Command "comment-empty-line-before": "never" works correctly, but I need to delete empty lines before double-slash comments
